### PR TITLE
[CID-3171] JMS selector does not work with fedmsg with selector "new …

### PR DESCRIPTION
**Pull request by ggallen**
…= 1".

### CODE REVIEW

**Comment by ggallen** [comment]
@scoheb , this change implements the UI changes we talked about. For FedMsg, the selector, messageType, and messageProperties fields are hidden where appropriate.

File: src/main/java/com/redhat/jenkins/plugins/ci/messaging/FedMsgMessagingWorker.java [hunk 1]
```diff
@@ -218,7 +218,6 @@ public void receive(String jobname, String selector, List<MsgCheck> checks, long
                 // Something other than an interrupt causes this.
                 // Unsubscribe, but stay in our loop and try to reconnect..
                 log.log(Level.WARNING, "JMS exception raised, going to re-subscribe for job '" + jobname + "'.", e);
-                log.log(Level.SEVERE, org.apache.commons.lang.exception.ExceptionUtils.getStackTrace(e));
```
Comment (scoheb) [inline src/main/java/com/redhat/jenkins/plugins/ci/messaging/FedMsgMessagingWorker.java]: I like this line...helpful sometimes. why do you want to remove?

**Comment by scoheb** [comment]
small comment. jelly stuff out of my comfort zone...

File: src/main/java/com/redhat/jenkins/plugins/ci/messaging/FedMsgMessagingWorker.java (hunk 1 above)
Comment (scoheb) [reply src/main/java/com/redhat/jenkins/plugins/ci/messaging/FedMsgMessagingWorker.java]: oh ok. Nevermind then.

**Pull request merged**